Implemented request-body data insertion through text-editor (#45)
hoppscotch/hopp-cli: README.md

$ hopp-cli post -c js http://example.com <myrequest.json
```

### Providing a Request Body via text-editor

> In addition to providing request body via `-b/ --body` flag and stdin,

Reviewer: ```suggestion
In addition to providing request body via `-b / --body` flag and stdin,
```

you can also use `-e / --editor` flag which opens default text-editor in your system.

**Example:**

```shell
$ hopp-cli post https://reqres.in/api/users/2 -c js -e
```

> It will preferrably open editor based on `$EDITOR` env.

Reviewer: ```suggestion
It will preferrably open editor based on `$EDITOR` environment variable.
```

**For example:**

> In your environment variable if `EDITOR=code` it will open VSCode for request-body input. Else, it will use default editor value based on OS.

Reviewer: ```suggestion
If the environment variable is`$EDITOR=code` it will open VSCode for request-body input. Else, it will use default editor value based on the OS.
```
